gems/pending/metadata/ScanProfile/modules/HostScanItemNteventlog.rb

```diff
@@ -72,7 +72,7 @@ def hostd_log_to_hashes(log, filter = nil)
         :level     => level,
         :source    => source,
         :message   => message,
-        :uid       => Digest::MD5.hexdigest("#{generated} #{name} #{level} #{source} #{message}")
+        :uid       => Rails.application.config.digest_class.hexdigest("#{generated} #{name} #{level} #{source} #{message}")
```

Comment:
@gmcculloug Please review.  Would it be possible to use an alternative to md5 here?